example/pubspec.yaml

```diff
@@ -43,8 +43,22 @@ dependencies:
   cupertino_icons: ^1.0.5
 
   # kevin. SDK plugins
-  kevin_flutter_core: ^1.0.2
-  kevin_flutter_in_app_payments: 1.0.0
+  kevin_flutter_core:
+    path: ../packages/kevin_flutter_core/kevin_flutter_core
+  kevin_flutter_in_app_payments:
+    path: ../packages/kevin_flutter_in_app_payments/kevin_flutter_in_app_payments
+
+dependency_overrides:
+  kevin_flutter_core:
```

Comment:
Is there any specific reason why you have same plugins added to `dependencies` and `dependency_overrides`?

Reply:
Since example app is stored together with libs, there is no point in depending on remote plugins, when we can depend on local ones. That makes it easier to change code in plugins and test it straight away.
Plugins themselves still depend on remote plugins, ex. `kevin_flutter_accounts` and `kevin_flutter_in_app_payments` depend on remote `kevin_flutter_core`. To override that behaviour, `dependency_overrides` is needed, so that every plugin depends on local package.

Reply:
Good point